components/butterfly/src/rumor/mod.rs

```diff
@@ -28,6 +28,11 @@ pub mod service;
 pub mod service_config;
 pub mod service_file;
 
+pub use self::election::Election;
```

Comment:
The exports still exist on disk, this is common in larger Rust code bases because it helps guide people through the public API of your modules. It allows you to place things in different files for logical organization as a developer but also exposing the module as a single logical unit for the public API consumer.

These are re-exported here because when you're dealing with Butterfly rumors as an API consumer you don't necessarily care that they're in their own sub modules, you just care about the 4 types of rumors and want to access them. It makes for less lines on import.

You'll also see re-exporting commonly used when your API asks for people to use a number of public types to interact with it in the form of a `prelude` module. I was actually one step away from including that until I noticed we were aliasing the rumors on import to avoid a namespace collision ;)